change/@graphitation-supermassive-cd7d08a1-748c-406f-ab44-003b47ed80b2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
This is technically a major change. At the very least it should be a minor. Same for all other packages. Structural changes shouldn't be published as a patch.